client/ayon_core/pipeline/farm/pyblish_functions.py

```diff
@@ -453,6 +469,50 @@ def prepare_representations(skeleton_data, exp_files, anatomy, aov_filter,
     return representations
 
 
+def _get_real_frames_to_render(frames):
+    """Returns list of frames that should be rendered.
+
+    Artists could want to selectively render only particular frames
+    """
+    frames_to_render = []
+    for frame in frames.split(","):
+        if "-" in frame:
+            splitted = frame.split("-")
+            frames_to_render.extend(
+                range(int(splitted[0]), int(splitted[1])+1))
+        else:
+            frames_to_render.append(frame)
+    return [str(frame_to_render) for frame_to_render in frames_to_render]
```

Comment:
Why converted to string?
```suggestion
    return [frame_to_render for frame_to_render in frames_to_render]
```